ios/RNTPTDocumentView.m

```diff
@@ -2236,48 +2238,68 @@ - (BOOL)rnt_documentViewControllerIsNavigationBarEnabled:(PTDocumentBaseViewCont
     return !self.hideTopAppNavBar;
 }
 
-- (NSArray<NSDictionary<NSString *, id> *> *)annotationDataForAnnotations:(NSArray<PTAnnot *> *)annotations pageNumber:(int)pageNumber pdfViewCtrl:(PTPDFViewCtrl *)pdfViewCtrl
+- (NSDictionary<NSString *, id> *)getAnnotationData:(PTAnnot *)annot pageNumber:(int)pageNumber pdfViewCtrl:(PTPDFViewCtrl *)pdfViewCtrl {
```

Comment:
Lock needed? Could someone ever use this method outside the "all" loop in the method below?

Reply:
Yes lock is needed when calling this function. Should I add a comment before this function, or move the lock from before for loop to this function?

Reply:
Actually thinking more, I think this is good as is. :)